src/Momento.Sdk/Auth/EnvMomentoTokenProvider.cs

```diff
@@ -4,10 +4,22 @@ namespace Momento.Sdk.Auth;
 using Momento.Sdk.Exceptions;
 using Momento.Sdk.Internal;
 
+/// <summary>
+/// Reads and parses a JWT token stored as an environment variable.
+/// </summary>
 public class EnvMomentoTokenProvider : ICredentialProvider
 {
+    /// <summary>
+    /// JWT provided by user
+    /// </summary>
     public string AuthToken { get; private set; }
-    public string ControlEndpoint {get; private set; }
+    /// <summary>
+    /// Control endpoint extracted from JWT
+    /// </summary>
+    public string ControlEndpoint { get; private set; }
+    /// <summary>
+    /// Cache endpoint extracted from JWT
```

Comment:
For these two I think you can just use inheritdoc to get the docs for whatever we put on the interface (@malandis @pgautier404  correct me if that sounds wrong.)